packages/device_info/lib/device_info.dart

```diff
@@ -116,6 +117,9 @@ class AndroidDeviceInfo {
   /// The type of build, like "user" or "eng".
   final String type;
 
+  ///false, if device is being simulated/emulated
+  final bool isPhysicalDevice;
```

Comment:
Blowing the dust off my bikeshedder hat: Technically you always run on a physical device. The question is whether we're running in an emulator. So, maybe `isEmulator` for `AndroidDeviceInfo` and `isSimulator` for `IosDeviceInfo`?

Reply:
Yeah, before I wanted to build my bikeshed, I already started shaving the yak ;-):

I thought about that too, but was more inclined to a consistent naming for the field, rather than forcing the user to differentiate which Os the code runs which kinda goes against the beautiful "one code - two platforms" idea of flutter. I am already not a big fan of the way iOsInfo and AndroidInfo are being differentiated (which is unavoidable I guess) and wanted to at least make this nomenclature more uniform. Especially because of the whole "simulator" vs. "emulator" distinction...

Reply:
Sounds good. Let's keep `isPhysicalDevice`.